area_printer.go

```diff
@@ -50,7 +50,19 @@ func (p AreaPrinter) WithCenter(b ...bool) *AreaPrinter {
 
 // SetWriter sets the writer for the AreaPrinter.
 func (p *AreaPrinter) SetWriter(writer io.Writer) {
+	cw, ok := writer.(cursor.Writer)
+	if ok {
+		p.SetCWriter(cw)
+	}
+}
 
+// SetWriter sets the writer for the AreaPrinter.
+func (p *AreaPrinter) SetCWriter(writer cursor.Writer) {
+	var newArea cursor.Area
+	if p.area == nil {
```

Comment:
the writer only gets set when the area is nil. otherwise the incoming writer is ignored. 

Is that a bug or a feature?